scan.py

```diff
@@ -110,6 +132,7 @@ def lambda_handler(event, context):
     print("Script starting at %s\n" %
           (start_time.strftime("%Y/%m/%d %H:%M:%S UTC")))
     s3_object = event_object(event)
+    verify_s3_object_version(s3_object)
```

Comment:
shouldn't we check it after sns_scan_results call?
basically at the end of the scan and before sending the event to SNS we should validate.
In case lambda takes some time and in between newer version appears then, we could abort, or may be we can just ignore as we are already sending the scanned version in the event to SNS.
or may be verify_s3_object_version doesn't abort and instead tag the latest version it fetched for scanning.